jb-yoda/src/main/java/life/jukebox/YodaSubscriberClient.java

```diff
@@ -0,0 +1,37 @@
+package life.jukebox.yoda;
+
+import com.google.cloud.ServiceOptions;
+import com.google.cloud.pubsub.v1.AckReplyConsumer;
+import com.google.cloud.pubsub.v1.MessageReceiver;
+import com.google.cloud.pubsub.v1.Subscriber;
+import com.google.pubsub.v1.PubsubMessage;
+import com.google.pubsub.v1.SubscriptionName;
+import java.util.concurrent.BlockingQueue;
+import java.util.concurrent.LinkedBlockingDeque;
+
+public class YodaSubscriberClient{
+  private static final String PROJECT_ID = getEnvironmentVariableOrDie("GCP_PROJECT_ID");
+  private static final String SUBSCRIPTION_ID = getEnvironmentVariableOrDie("PUBSUB_SUB_JUKEBOX_CREATED_YODA");
+
+    private static String getEnvironmentVariableOrDie (String environmentKey) {
```

Comment:
Did you run this through a linter? Extra indentation for this method..
I heavily suggest using a linter (like [this](https://github.com/checkstyle/checkstyle))